### DISCUSSION TITLE
Support for Android 4.2.2

### DISCUSSION THREAD

**Discussion by TheMercenary333** [Docile-Alligator/Infinity-For-Reddit#1317] [Q&A]
Is there any chance to support android 4.2.2 (API 17) armv7 device?

**Comment by Wladefant** [comment]
Well, if you mean use an earlier version of the android API instead of the current one. Definitely no. Sorry.
Try to fix some bugs that are occuring in earlier Android. Well, report them and we will try to fix them, if they are not too specific to your android version. 🤷‍♂️
